Updates to Introduction for methods section (#947)
greenelab/covid19-review: content/60.methods.md

This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated viruses [@arxiv:2004.10706] and contained {{cord19_total_pubs}} manuscripts as of {{cord19_date_pretty}}.
Additional curation by CoronaCentral [@doi:10.1101/2020.12.21.423860] has produced, at present, a set of over 150,000 publications particularly relevant to COVID-19 and these closely related viruses.
Thus, any effort to synthesize, summarize, and contextualize COVID-19 research will face a vast corpus of potentially relevant material.

**Change over time in the number of publications in the CORD-19 dataset.**
As of {{cord19_date_pretty}}, there are {{cord19_total_pubs}} in the CORD-19 dataset.
The first release, on March 16, 2020, contained 28,000 manuscripts on topics relevant to SARS-CoV-2 and related coronaviruses [@arxiv:2004.10706].
Since then, these articles have continued to proliferate (left), with both traditionally published and preprint manuscripts in the corpus (right).
At present, it contains {{cord19_total_preprints}} preprints from _arXiv_, _bioRxiv_, and _medRxiv_.

> While not all of the manuscripts are focused explicitly on SARS-CoV-2 or COVID-19, this corpus is likely to contain all or most manuscripts relevant to writing a literature review, which requires assessing both emerging and prior research.

Reviewer: In the future, one could imagine examining the fraction of papers that we cite that are in CORD.

We extended Manubot to support directly citing any of the hundreds of Compact Uniform Resource Identifier registered with <https://identifiers.org/>, including the `clinicaltrials` identifier.
This extension enables citing a trial as `clinicaltrials:NCT04280705` instead of `https://clinicaltrials.gov/ct2/show/NCT04280705`.

Another challenge that emerged was that, because of the large number of citations used in this manuscript and the fast-moving nature of COVID-19 research, keeping track of retractions, corrections, and notices of concern became a priority.
A new plugin was added to Manubot to support "smart citations" in the HTML build of manuscripts.
The plugin uses the [Scite](https://scite.ai/) [@doi:10.1101/2021.03.15.435418] service to display a badge below any citation with a DOI.
The badge contains a set of icons and numbers that indicate how many times that source has been mentioned, supported, or disputed and whether there have been any important editorial notices, such as retractions or corrections.

> Using this, we were able to identify references that needed to be reevaluated.

Reviewer: In the idea world, we'd cite a retracted paper here so that folks could see the rendering but that won't work with ACM's template. What about citing a retracted paper and then putting a link to the instance in the manubot html? Maybe this is too self referential.

In addition, we expanded the export formats to include partial LaTeX support via Pandoc.
Pandoc converts the markdown content for an individual section to tex and the Citation Style Language JSON, which contains reference metadata generated by Manubot, to BibTeX.
We customized a LaTeX template and reformat the Manubot metadata, such as authors and their affiliations, for the LaTeX template.
Because LaTeX is required for manuscript submission in many fields, automating most of the process of converting markdown to a submission-friendly format expands Manubot's potential user base.
Manubot users can write in the simple markdown format, render the manuscript in regularly-updated pdf or interactive HTML formats, and export the manuscript in docx or tex and BibTeX for submission to traditional publishers, taking full advantage of Pandoc's powerful document conversion capabilities.

### RESULTS

#### Recruitment and Manuscript Development

![
**Project growth over time.**
The number of authors, word count, and number of references have all grown dramatically from when the project began on March 20, 2020.

> As of April 30, 2021, there are 49 authors, 1,428 references, and 131,949 words in the documents that make up the project.

Author: ```suggestion
As of April 30, 2021, there are 51 authors, 1,428 references, and 131,949 words in the documents that make up the project.
```

Additionally, new contributors have continued to join even as the project begins its second year.

(Dot plot of contributions will go here) <!--To Do: Just adding a comment to add the caption and reference to figure here and also about in "Figure with dots". Also, I'm only counting 41 authors here, whereas 49 are mentioned earlier and later. Is that just because some folks contributed but did not use GitHub? If so, should note that.-->

In order to make the project accessible to individuals from a range of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request [@url:https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md; @url:https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
Additionally, the framework for evaluating literature was converted into issue templates to simplify the review of new articles.

> Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.<!-- To Do: Number of paper issue and number of unique contributors?-->

Reviewer: Agree - at least issue number would be helpful, perhaps broken out by tags if enough got categorized.

> All seven manuscripts associated with this project will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.

Reviewer: ```suggestion
All seven manuscripts associated with this project except for this one will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.
```

Author: This is an eighth manuscript! (too many manuscripts)
```suggestion
The seven manuscripts produced by the consortium (beyond this one) will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews so that they can continue to be updated as more information becomes available.
```

Currently, three manuscripts have been submitted, with one accepted for publication [@individual-nutraceuticals] and two under revision [@individual-pathogenesis; @individual-pharmaceuticals].
The remaining four manuscripts are in preparation.
These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@individual-pharmaceuticals], nutraceutical therapeutics [@individual-nutraceuticals], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing disease transmission and outcomes [@individual-inequality].

> To date, 49 authors are associated with the consortium (Figure @fig:projectstats).<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->

Reviewer: Is this still true? At least Vince got added to this paper for scite so we should make sure he rolls into the consortium.

Author: Daniel did too so we are either at 50 or 51, I have to check whether the Vince authorship merge was one of the commits used to generate this stat!

Author: OK Vince is 49, Daniel is 50, but I haven't merged in one other author so it's actually 51.
```suggestion
To date, 51 authors are associated with the consortium (Figure @fig:projectstats).<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->
```

Thus, the consortium is expected to be successful in providing a venue for researchers across all career stages to continue investigating and publishing at a time when many biomedical researchers were unable to access their laboratory facilities.

#### Using Manubot to Investigate COVID-19

Data were integrated into the manuscripts from several sources.
Data about worldwide cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series] were read using a Python script.
Similarly, the clinical trials statistics and figure were generated based on data from the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
The evolution of this figure over time is shown in Figure @fig:ebm-trials.
Information about vaccine distribution was extracted from Our World In Data [@url:https://github.com/owid/covid-19-data].

> Figure 1 (above) also uses this approach to dynamically integrate data directly from the CORD-19 dataset [@arxiv:2004.10706].<!--To Do: Flow chart of data integration? We could have a summary figure showing all of the external data sources that are integrated into the manuscript. We have icons for MSSM reviews, JHU data, etc. at the top. That flows into a GitHub repo, which also takes input from all the contributors. Then the output of the repo is the manuscript and other CI artifacts.-->

Reviewer: Time probably doesn't exist, but I do like this figure idea. It should definitely be in a presentation if accepted at the conference.

![
**Change in the COVID-19 clinical trials figure over time.**
When we first produced this figure on July 7, 2020, there were 3,733 clinial trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
As of {{ebm_date_pretty}}, it contains {{ebm_all_trials}}.
We were also able to easily reconfigure the figure prior to journal submission to emphasize interventional trials based on the recommendation of a collaborator who is a clinician.
This figure is included in an analysis of pharmaceutical development efforts during COVID-19 [@individual-pharmaceuticals].
]((images/ebmdatalab-trials-original.png){{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

Manubot's bibliographic management capabilities ended up being critical because the amount of relevant literature published far outstripped what we had anticipated at the beginning of the project.

> As of April 30, 2021, there are 1,428 references (Figure @fig:projectstats).

Reviewer: ```suggestion
As of April 30, 2021, there were 1,428 references (Figure @fig:projectstats).
```

> ### ABSTRACT

Author: Does ACM have a word limit on abstracts? I couldn't find it. This is 363 words which seems too long, but if they don't say otherwise...

Reviewer 2: The sample manuscript doesn't state a word limit either https://github.com/greenelab/covid19-review/blob/c3bf3f4245cc4fec4c37f349184015e1eafdacfc/build/assets/acmart-master/samples/sample-sigconf.tex#L169

Additional curation by CoronaCentral [@doi:10.1101/2020.12.21.423860] has produced, at present, a set of over 150,000 publications particularly relevant to COVID-19 and these closely related viruses.
Thus, any effort to synthesize, summarize, and contextualize COVID-19 research will be faced with a vast corpus of potentially relevant material.

<!--![
**Change over time in the number of publications in the CORD-19 dataset.**
As of {{cord19_date_pretty}}, there are {{cord19_total_pubs}} in the CORD-19 dataset.
The first release, on March 16, 2020, contained 28,000 manuscripts on topics relevant to SARS-CoV-2 and related coronaviruses [@arxiv:2004.10706].
Since then, the resource has continued to grow rapidly (left), with both traditionally published and preprint manuscripts in the corpus (right).
At present, it contains {{cord19_total_preprints}} preprints from _arXiv_, _bioRxiv_, and _medRxiv_.
While not all of the manuscripts are specifically focused on SARS-CoV-2 or COVID-19, this corpus is likely to contain all or most manuscripts that would be explored in developing a literature review, which requires assessing both emerging and prior research.

> ]({{cord19-growth}} "Global COVID-19 deaths"){#fig:cord19-growth secno=1}

Author: Thank you for catching this! 🤦‍♀️

-->
<!--To Do: Possible viz of change in clinical trials as well?-->

With information being produced rapidly through both traditional publishing venues and preprint servers, some papers that are published face scrutiny after their initial release.
Concerns have been raised that the number of COVID-19 papers being retracted may be higher, or potentially much higher, than is typical, although a thorough investigation of this question will not be possible until more time has elapsed [@doi:10.1080/08989621.2020.1782203; @doi:10.1080/08989621.2020.1793675].

> Other papers are updated with corrections or expressions of concern [@doi:10.1080/08989621.2020.1793675; @url:https://retractionwatch.com/retracted-coronavirus-covid-19-papers].

Author: Oh yikes, I had a few URLs in this one! I'm happy to just manually fix things if that's the easiest way to do it.

> Other papers are updated with corrections or expressions of concern [@doi:10.1080/08989621.2020.1793675; @url:https://retractionwatch.com/retracted-coronavirus-covid-19-papers].

Reviewer 2: I'm not sure what the best approach is.  Check the latest PDF build in https://github.com/greenelab/covid19-review/pull/943/#issuecomment-830197430 to see how these will render in the reference list.  Considering our time crunch, this may be low priority.  Let's take care of the other content first.

These include both preprints and papers that are published in more traditional venues [@url:https://retractionwatch.com/retracted-coronavirus-covid-19-papers; @url:https://asapbio.org/preprints-and-covid-19].
Preprints provide a venue for scientists to release findings rapidly, but have both the advantage and disadvantage of making research available before it has undergone the peer review process.
However, some traditional publishing venues have also fast-tracked COVID-19 through peer review, leading to questions about whether this research is being held to the usual standards for publication [@doi:10.1111/bioe.12772].
Therefore, monitoring the COVID-19 literature requires not only digesting the high volume of information being released, but also critically evaluating it and/or monitoring for subsequent adjustments.

Because of the fast-moving nature of the topic, many efforts to summarize and synthesize COVID-19 literature have been undertaken. 

> These efforts include newsletters [@url:https://depts.washington.edu/pandemicalliance/covid-19-literature-report/latest-reports/; @doi:10.1080/10872981.2020.1770562], web portals (such as [@url:https://outbreaksci.prereview.org/] or the now-defunct http://covidpreprints.com/, which was described in [@url:https://asapbio.org/preprints-and-covid-19]), comments on preprint servers [@doi:10.1038/s41577-020-0319-0; @url:https://disqus.com/by/sinaiimmunologyreviewproject/], and even a journal [@url:https://rapidreviewscovid19.mitpress.mit.edu/].

Author: ```suggestion
These efforts include newsletters [@url:https://depts.washington.edu/pandemicalliance/covid-19-literature-report/latest-reports/; @doi:10.1080/10872981.2020.1770562], web portals (such as [@url:https://outbreaksci.prereview.org; @doi:10.1126/science.abc7839] or the now-defunct http://covidpreprints.com/, which was described in [@url:https://asapbio.org/preprints-and-covid-19]), comments on preprint servers [@doi:10.1038/s41577-020-0319-0; @url:https://disqus.com/by/sinaiimmunologyreviewproject/], and even a journal [@url:https://rapidreviewscovid19.mitpress.mit.edu/].
```

However, the explosive rate of publication presents challenges for such efforts, many of which are no longer publishing summaries.
Similarly, many literature reviews have been written on the available COVID-19 literature [@doi:10.1016/j.molmed.2020.02.008; @doi:10.1016/j.immuni.2020.05.002; @doi:10.1126/scitranslmed.abc1931; @doi:10.1016/j.immuni.2020.05.002; @doi:10.1001/jama.2020.6019; @doi:10.1038/d41591-020-00026-w; @doi:10.1001/jama.2020.12839].

> However, static reviews quickly become outdated as new research is released or existing research is retracted or superseded

Author: ```suggestion
However, static reviews quickly become outdated as new research is released or existing research is retracted or superseded; one example is a review that reviewed topics in COVID-19 research including vaccine development [@doi:10.1001/jama.2020.12839].
This review was published on July 10, 2020, four days before Moderna released the surprisingly promising results of their phase 1 trial [@doi:10.1056/NEJMoa2022483] that changed expectations surrounding vaccines.
```

Therefore, the COVID-19 publishing climate presented a challenge where curation of the literature by a diverse group of experts in a format that could respond quickly to high-volume, high-velocity information was desirable.

We therefore sought to develop a platform for scientific discussion and collaboration around COVID-19 by adapting open publishing infrastructure to accommodate the scale of the COVID-19 publishing boom.
Recent advances in open publishing have created an infrastructure that facilitates distributed, version-controlled collaboration on manuscripts [@doi:10.1371/journal.pcbi.1007128].<!--To Do: possibly cite some other efforts here-->
Manubot [@doi:10.1371/journal.pcbi.1007128] is a collaborative framework developed to adapt open-source software development techniques and version control for manuscript writing.
With Manubot, manuscripts are managed and maintained using GitHub, a popular, online version control interface that also provides the infrastructure via continuous integration (CI) to incorporate code into the manuscript building process to allow, for example, figures to be continuously updated based on an external data set.

> This open-publishing platform has been used to develop large-scale collaborative efforts such as a review of developments in deep learning [@doi:10.1098/rsif.2017.0387].

Reviewer 2: This is not essential but https://doi.org/10.1080/08989621.2020.1779591 is another example large scale collaboration.

Author: ```suggestion
This open-publishing platform has been used to develop large-scale collaborative efforts such as a review of developments in deep learning [@doi:10.1098/rsif.2017.0387] and a re-evaluation of the role of authorship in modern collaborations [@doi:10.1080/08989621.2020.1779591].
```

Collaboration via massively open online papers (MOOPs) has been identified as a strategy for promoting inclusion and interdisciplinary thought [@doi:10.5334/kula.63].
Manubot is an ideal platform for the analysis of COVID-19 literature because it facilitates the automatic integration of new data through CI.
However, the Manubot workflow can appear intimidating to contributors who are not well-versed in git [@doi:10.5334/kula.63].

> The synthesis and discussion of the emerging literature by biomedical scientists and clinicians is imperative to a robust interpretation of COVID-19 research, but in biology, such efforts often rely on WYSIWYG tools such as Google Docs despite the significant limitations of these platforms in the face of excessive publication.

Reviewer 2: ```suggestion
The synthesis and discussion of the emerging literature by biomedical scientists and clinicians is imperative to a robust interpretation of COVID-19 research, but in biology, such efforts often rely on What You See Is What You Get tools such as Google Docs despite the significant limitations of these platforms in the face of excessive publication.
```

One option was to catalog articles of interest as issues in the GitHub repository.
We developed a standardized set of questions for contributors to consider when evaluating an article following a framework often used for assessing medical literature.
This approach emphasizes examining the methods used, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].
Contributors were also invited to contribute or edit text using GitHub's pull request system.
These contributions were not strictly defined and could range from minor corrections to punctuation and grammar to large-scale additions of text.
Each pull request was reviewed and approved by at least one other contributor before being merged into the main branch.
We sought to tag potential reviewers based on the introductions they had contributed in order to encourage participation.
Emphasizing the use of issues and pull requests was designed to encourage authors with and without git experience to discuss papers and provide feedback (both formal and informal) on proposed text additions or changes.

> We also used gitter (@url:https://gitter.im) to promote informal questions and sharing of information among collaborators.

Reviewer 3: ```suggestion
We also used the Gitter chat platform (@url:https://gitter.im) to promote informal questions and sharing of information among collaborators.
```

Manubot uses the Zotero translation server (https://www.zotero.org and https://github.com/zotero/translation-server) to extract metadata for some types of citations.
However, Zotero did not support clinical trial identifiers and could not extract relevant metadata from the clinical trial's URL.
In order to enable Manubot to pull metadata associated with clinical trials based on their identifiers, we added Zotero support for these identifiers.
Other researchers identified the same need [@url:https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry; @url:https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org].
To implement this feature, we query clinicaltrials.gov to retrieve XML metadata associated with each identifier using JavaScript [@url:https://github.com/zotero/translators/pull/2153].
Then, when Manubot requests clinical trial metadata from the Zotero translation server, it receives a more informative response that includes the trial sponsors, responsible investigators, title, and summary.

> We extended Manubot to support directly citing any of the hundreds of Compact Uniform Resource Identifier registered with <https://identifiers.org/>, including the `clinicaltrials` identifier.

Reviewer 3: ```suggestion
We extended Manubot to support directly citing any of the hundreds of Compact Uniform Resource Identifiers registered with <https://identifiers.org/>, including the `clinicaltrials` identifier.
```

Using this, we were able to identify references that needed to be reevaluated.
This addition was invaluable given the nature of the project, where we were disseminating rapidly evolving information of great consequence from over a thousand different sources.
The badges also allow readers to ascertain a rough evaluation of the reliability of cited sources at a glance.

Because most collaborators were writing and editing text through the GitHub website rather than in a local text editor, we also needed to add spell-checking functionalities to Manubot.
We integrated an existing Pandoc (https://pandoc.org/) spell-check extension with AppVeyor CI to automatically post spelling errors as comments in a GitHub pull request.
The comment reported both unique misspelled words and all locations in which those spelling errors were detected.
Project maintainers created and updated a custom dictionary to ignore over 1,500 scientific and technical terms that are not common English words.

> Spell-checking also helped standardize the writing style across dozesn of authors by detecting British English spelling.

Reviewer 3: ```suggestion
Spell-checking also helped standardize the writing style across dozens of authors by detecting British English spelling.
```

The actual spell-checking was implemented using GNU Aspell (http://aspell.net/) and the Pandoc spellcheck filter [@url:https://github.com/pandoc/lua-filters/tree/master/spellcheck].
The filter enables checking only the manuscript text, ignoring URLs and formatting text.
<!-- To Do: Acknowledge David Nicholson for the suggestion and edit to report locations of spelling errors -->

Manubot can render a manuscript in several formats that serve different purposes.

> Prior to this project, Manubot was able to convert the markdown-formatted manuscript to HTML, pdf, and docx formats.

Reviewer 3: ```suggestion
Prior to this project, Manubot was able to convert the markdown-formatted manuscript to HTML, PDF, and DOCX formats.
```

> For the current project, we expanded this functionality to allow for individual sections of the manuscript to be rendered as separate docx files.

Reviewer 3: ```suggestion
For the current project, we expanded this functionality to allow for individual sections of the manuscript to be rendered as separate DOCX files.
```

### RESULTS

#### Recruitment and Manuscript Development

![
**Project growth over time.**
The number of authors, word count, and number of references have all grown dramatically from when the project began on March 20, 2020.
As of April 30, 2021, there are 51 authors, 1,428 references, and 131,949 words in the documents that make up the project.
For a period of time in summer 2020, a large appendix was erroneously duplicated, leading to the apparent spike that was later removed.
]((images/manuscript_stats.png)("Project stats over time"){#fig:projectstats secno=1}

We received a large amount of interest from the broad community, beginning with coverage of the project by _Nature Toolbox_ [@doi:10.1038/d41586-020-00916-6] and an associated tweet about the project on April 1, 2020 [@url:https://twitter.com/j_perkel/status/1245454628235309057] (Figure @projectstats).
Because the GitHub issues and comment systems are relatively similar to other common web activities, we found that authors were able to learn these tools fairly quickly.

> Similarly, the gitter chat also presented a low barrier to entry.

Reviewer 3: ```suggestion
Similarly, the Gitter chat also presented a low barrier to entry.
```

To date, 51 authors are associated with the consortium (Figure @fig:projectstats).<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->
Efforts to integrate with existing projects providing support for undergraduate students during COVID-19 were also successful.
We collaborated with the Immunology Institute at the Mount Sinai School of Medicine to incorporate summaries written by their students, post-docs, and faculty [@url:https://github.com/ismms-himc/covid-19_sinai_reviews; @doi:10.1038/s41577-020-0319-0].
Additionally, two of the consortium authors were undergraduate students recruited through the American Physician Scientist Association's Virtual Summer Research Program.
Thus, the consortium is expected to be successful in providing a venue for researchers across all career stages to continue investigating and publishing at a time when many biomedical researchers were unable to access their laboratory facilities.

> #### Using Manubot to Investigate COVID-19

Reviewer 3: This heading doesn't really reflect the following paragraph... Maybe `Using Manubot to Collate Public Information Sources` or similar?

![
**Change in the COVID-19 clinical trials figure over time.**
When we first produced this figure on July 7, 2020, there were 3,733 clinial trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
As of {{ebm_date_pretty}}, it contains {{ebm_all_trials}}.
We were also able to easily reconfigure the figure prior to journal submission to emphasize interventional trials based on the recommendation of a collaborator who is a clinician.
This figure is included in an analysis of pharmaceutical development efforts during COVID-19 [@individual-pharmaceuticals].
]((images/ebmdatalab-trials-original.png){{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

Manubot's bibliographic management capabilities ended up being critical because the amount of relevant literature published far outstripped what we had anticipated at the beginning of the project.
As of April 30, 2021, there were 1,428 references (Figure @fig:projectstats).

> The scite plugin provided a way to visually inspect the reference list to identify possible references of concern.

Reviewer 3: ```suggestion
The Scite plugin provided a way to visually inspect the reference list to identify possible references of concern.
```

### DISCUSSION

The current project was managed through GitHub (https://github.com/greenelab/covid19-review) using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate a version of the manuscript online [@url:https://greenelab.github.io/covid19-review].
The Manubot framework facilitated a massive collaborative review on an urgent topic.
This project demonstrates that Manubot can be applied to projects where not all collaborators have expertise or even experience working with version control pipelines.
Through the development of cyberinfrastructure both for training novice users to interact with GitHub and to simplify the workflows to allow them to receive many of the benefits of What You See Is What You Get platforms such as Google Docs, we were able to adapt a powerful open publishing tool to harness the domain expertise of a large group of non-technical users and to respond to the flood of COVID-19 publications.

> While Manubot manuscripts are written in markdown, they can be rendered in several formats that provide different advantages.

Reviewer 3: ```suggestion
While Manubot manuscripts are written in markdown, they can be rendered in several formats that provide different advantages.
```
```suggestion
While Manubot manuscripts are written in Markdown, they can be rendered in several formats that provide different advantages.
```

> For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, docx, and now, LaTeX.

Reviewer 3: ```suggestion
For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, DOCX, and now, LaTeX.
```

The HTML manuscript format offers several advantages over a static PDF to harmonize available resources that we were able to apply to specific problems of COVID-19.

> The integration of scite into the HTML build makes references more manageable by visually representing whether their results are contested or whether they have been corrected or retracted.

Reviewer 3: ```suggestion
The integration of Scite into the HTML build makes references more manageable by visually representing whether their results are contested or whether they have been corrected or retracted.
```

Cross-referencing different pieces of the manuscript, such as cited preprints with reviews stored in an appendix, is another dynamic option presented by HTML.

> Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate docx outputs was expanded to allow users to generate docx files containing only a section of the manuscript.

Reviewer 3: ```suggestion
Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate DOCX outputs was expanded to allow users to generate DOCX files containing only a section of the manuscript.
```

In our case, where the full project is nearly 100,000 words, this allows individual pieces to be shared widely.

> Finally, the addition of LaTeX output is useful for researchers from computational fields who submit papers in tex format and removes the step of reformatting markdown prior to submission. 

Reviewer 3: ```suggestion
Finally, the addition of LaTeX output is useful for researchers from computational fields who submit papers in TeX format and removes the step of reformatting Markdown prior to submission. 
```

### METHODS

#### Contributor Recruitment and Roles

A preliminary requirement for this undertaking was to establish Manubot as a platform accessible to researchers with limited experience working with git, as is common in biology and medicine, where version control is not typically emphasized [@doi:10.1109/SE4Science.2017.11; @doi:10.1177/2515245918754826; @doi:10.1186/1751-0473-8-7].
Contributors were recruited by word of mouth and on Twitter, and we sought out opportunities to integrate existing efforts to train early-career researchers (ECRs).
We invited potential collaborators to contribute a short introduction on a GitHub issue in order to collect information about who was involved and provide an introduction to working with GitHub issues.
Interested participants were encouraged to contribute in a number of ways.
One option was to catalog articles of interest as issues in the GitHub repository.

> We developed a standardized set of questions for contributors to consider when evaluating an article following a framework often used for assessing medical literature.

Reviewer 2: Are we including these in an appendix?  Are we even allowed to have an appendix?

Author: I'm assuming no appendices are allowed, so I just linked them in the results!

Contributors were also invited to contribute or edit text using GitHub's pull request system.
These contributions were not strictly defined, and could range from minor corrections to punctuation and grammar to large-scale additions of text.
Each pull request was reviewed and approved by at least one other contributor before being merged into the main branch.
We sought to tag potential reviewers based on the introductions they had contributed in order to encourage participation.
The emphasis on issues and pull requests was designed to encourage authors to discuss papers in the issues and to provide feedback (both formal and informal) on proposed text additions or changes.
We also used gitter [@url:https://gitter.im] to promote informal questions and sharing of information among collaborators.

#### Applying Manubot's Existing Capabilities to the Challenges of COVID-19 

Because of the ever-evolving nature of the COVID-19 crisis, many of the figures and text proposed by subject matter contributors would have quickly become outdated.

> To address this concern, Manubot and GitHub's CI features were used to create figures that integrated online data sources to respond to changes in the COVID-19 pandemic over time.

Reviewer 2: I see this as a new feature.  Previously Manubot could only include static images, including versioned static images from an external GitHub repository.

Author: Oh wow! I didn't realize this was the first time this was done. I'll move it down in my next commit.

The template variables also include versioned URLs to the dynamically updated figures.
The JSON files and figures are stored in the `external-resources` branch of the manuscript's GitHub repository, which acts as versioned storage.
The GitHub Actions workflow automatically adds and commits the new JSON files and figures to the `external-resources` branch every time it runs, and Manubot uses the latest version of these resources when it builds the manuscript.
For figures, data are then plotted using Matplotlib [@doi:10.1109/MCSE.2007.55] in Python and pushed to a branch where they can be read into the main manuscript.

This approach also allowed us to minimize the demand on authors to curate bibliographic resources.
Manubot provides the functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
The author can insert a citation in-line using a format such as {% raw %}`[@doi:10.1371/journal.pcbi.1007128]`{% endraw %}.

> Manubot then obtains reference metadata, exports the citations as Citation Style Language (CSL) JSON Data Items, which is an open standard, and renders the bibliographic information needed to generate the references section [@doi:10.1371/journal.pcbi.1007128].

Reviewer 2: ```suggestion
Manubot then obtains reference metadata, exports the citations as Citation Style Language JSON Data Items, which is an open standard, and renders the bibliographic information needed to generate the references section [@doi:10.1371/journal.pcbi.1007128].
```

A new plugin was added to Manubot to support "smart citations" in the HTML build of manuscripts.
The plugin uses the [Scite](https://scite.ai/) service to display a badge below any citation with a DOI.
The badge contains a set of icons and numbers that indicate how many times that source has been mentioned, supported, or disputed, and whether there have been any important editorial notices, such as retractions or corrections.
Using this, we were able to quickly identify references that needed to be checked again since the time they had been added.
This was invaluable given the nature of the project, where we were disseminating rapidly evolving information of great consequence from over a thousand different sources.
The badges also allow readers to roughly evaluate the reliability of cited sources at a glance.

Because in this implementation of Manubot, most collaborators were writing and editing text through the GitHub website rather than in a local text editor, we also needed to add spell-checking functionalities to Manubot.

> <!--To Do: Describe -- perhaps @agitter can lead this?-->

Author: @agitter this is an area where I am super not knowledgeable, in case you're looking for a place to fill in text!

Reviewer 2: I can write about this.

Manubot provides the advantage of allowing a manuscript to be rendered in several formats that serve different purposes, and the current project extended these options.
Docx is a necessary format for a biological collaboration where authors are typically not working in LaTeX.
<!--To Do: continue expanding to describe docx updates and possibly LaTeX updates-->

> <!-- To Do: Also, if we end up automating anything to do with the Manubot-to-LaTeX or bibtex workflow while trying to submit this paper, we should describe that too, as well as the udpates for the Microsoft Word crowd to generating the complete review manuscript along with the individual manuscripts reviewing specific topics. <!-- To Do: reference individual manuscripts -->

Author: @mprobson if you want to add some info about LaTeX, this would be the place! I'll work on expanding this paragraph as well about the docx stuff (as @agitter mentioned above)

Reviewer 2: I'm writing something for this

![
**Change in the COVID-19 clinical trials figure over time.**
When we first produced this figure on July 7, 2020, there were 3,733 clincial trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
As of {{ebm_date_pretty}}, it contains {{ebm_all_trials}}.
We were also able to easily reconfigure the figure prior to journal submission to emphasize interventional trials based on the recommendation of a collaborator who is a clinician.
This figure is included in an analysis of pharmaceutical development efforts during COVID-19 [@individual-pharmaceuticals].
]((images/ebmdatalab-trials-original.png){{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

<!--To Do: Discussion of the absurd number of references?-->

The new features required for the COVID-19 project are now included in Manubot's rootstock.
Manubot and Zotero now support citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899], and the scite integration and spell-checking functionalities have been integrated into the current release of Manubot <!--To Do: Link-->.

> <!--To Do: have the individual builds been pushed up to rootstock?-->

Author: A question for you @agitter!

Another issue that emerged was the need for a standardized way to cite clinical trials.
Clinical trials that are registered with clinicaltrials.gov [@url:https://clinicaltrials.gov] receive a unique clinical trial identifier.
Because clinical trials are registered long before results are available in manuscript form, it was important to this project to be able to refer to the clinical trial identifiers associated with a large number of relevant trials (Figure @fig:ebm-trials).
Manubot uses Zotero [@url:https://www.zotero.org] to extract metadata for some types of citations, but Zotero did not support clinical trial identifiers.
In order to enable Manubot to pull metadata associated with clinical trials based on their identifiers, we needed to develop Zotero support for these identifiers.
The need for this feature to be enabled in Zotero had also been identified by other researchers [@url:https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry; @url:https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org].

> This update was achieved through the implementation of a query of clinicaltrials.gov to retrieve XML metadata associated with each identifier using JavaScript [@url:https://github.com/zotero/translators/pull/2153]. 

Reviewer 2: @rdvelazquez I pushed a little more detail to this paragraph.  If you have time, could you please double-check it again?

The badges also allow readers to roughly evaluate the reliability of cited sources at a glance.

Because in this implementation of Manubot, most collaborators were writing and editing text through the GitHub website rather than in a local text editor, we also needed to add spell-checking functionalities to Manubot.
<!--To Do: Describe -- perhaps @agitter can lead this?-->

Manubot provides the advantage of allowing a manuscript to be rendered in several formats that serve different purposes, and the current project extended these options.
Prior to this project, Manubot was able to convert the markdown-formatted manuscript to HTML, pdf, and docx formats.
For the current project, we expanded this functionality to allow for individual sections of the manuscript to be rendered as separate docx files.
This development was necessary because the manuscript grew so large that it needed to be split into seven separate papers for submission.
Similarly, we expanded the range of possible export formats to include LaTeX.
Because LaTeX is used for manuscript submission in many fields, automating the process of converting markdown to a submission-friendly format is desirable. 

> <!-- To Do: Also, if we end up automating anything to do with the Manubot-to-LaTeX or bibtex workflow while trying to submit this paper, we should describe that too, as well as the udpates for the Microsoft Word crowd to generating the complete review manuscript along with the individual manuscripts reviewing specific topics.

Author: @mprobson

We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request [@url:https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md; @url:https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
Additionally, the framework for evaluating literature was converted into issue templates to simplify review of new articles.
Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.
<!-- To Do: Number of paper issue and number of unique contributors?-->

> Currently, three manuscripts associated with this project have been submitted to _mSystems_, with one accepted for publication [@arxiv:2102.02250] and two under revision [@arxiv:2103.02723; @arxiv:2102.01521].

Reviewer 2: ```suggestion
Currently, three manuscripts associated with this project have been submitted to _mSystems_, with one accepted for publication [@individual-nutraceuticals] and two under revision [@individual-pathogenesis; @individual-pharmaceuticals].
```

Do we talk anywhere about how we have one mega-review on all topics as an HTML document and can programmatically split different sections into standalone manuscripts?  That's a pretty cool benefit of CI and automation.

Author: I will add that! Thank you for pointing it out.

An additional four manuscripts are in preparation.

> All seven will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they will continue to be updated as more information becomes available.

Reviewer 2: ```suggestion
All seven will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.
```

"Will" seemed a bit strong to me given how much work it takes to get the first versions out.

> These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@arxiv:2102.01521] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@arxiv:2103.02723], nutraceutical therapeutics [@arxiv:2102.02250], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing variable effects on patients [@individual-inequality].<!--To Do: add these additional individual citations-->

Reviewer 2: ```suggestion
These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@individual-pharmaceuticals], nutraceutical therapeutics [@individual-nutraceuticals], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing variable effects on patients [@individual-inequality].<!--To Do: add these additional individual citations-->
```

Using the aliases will let us easily update all of these cross-references to the DOIs later.

> To date, [number] authors are associated with the consortium.<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->

Reviewer 2: This could a place to cross-reference growth in the number of authors if we plot that.

Efforts to integrate with existing projects providing support for undergraduate students during COVID-19 were also successful.
Appendix A contains summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine [@url:https://github.com/ismms-himc/covid-19_sinai_reviews; @doi:10.1038/s41577-020-0319-0].
Additionally, two of the consortium authors were undergraduate students recruited through the American Physician Scientist Association's Virtual Summer Research Program [@url:https://www.physicianscientists.org/page/summer-research-pilot-program].
Thus, the consortium is expected to be successful in providing a venue for researchers across all career stages to continue investigating and publishing at a time when many biomedical researchers were unable to access their laboratory facilities.

#### Integration of COVID-19 Resources with Manubot

Data were integrated into the manuscripts from a number of sources.

> The workflow file is available from <https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml> and the scripts are available from <https://github.com/greenelab/covid19-review/tree/external-resources>.

Reviewer 2: I would stick our code availability in METHODS.

Author: Excellent, I'll move it up in my next commit!

The evolution of this figure over time is shown in Figure @fig:ebm-trials.
Information about vaccine distribution was extracted from Our World In Data [@url:https://github.com/owid/covid-19-data].
Figure 1 (above) also uses this approach to dynamically integrate data directly from the CORD-19 dataset [@arxiv:2004.10706].<!--To Do: Flow chart of data integration? We could have a summary figure showing all of the external data sources that are integrated into the manuscript. We have icons for MSSM reviews, JHU data, etc. at the top. That flows into a GitHub repo, which also takes input from all the contributors. Then the output of the repo is the manuscript and other CI artifacts.-->

![
**Change in the COVID-19 clinical trials figure over time.**
When we first produced this figure on July 7, 2020, there were 3,733 clincial trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
As of {{ebm_date_pretty}}, it contains {{ebm_all_trials}}.
We were also able to easily reconfigure the figure prior to journal submission to emphasize interventional trials based on the recommendation of a collaborator who is a clinician.

> This figure is included in an analysis of pharmaceutical development efforts during COVID-19 [@arxiv:2103.02723].

Reviewer 2: ```suggestion
This figure is included in an analysis of pharmaceutical development efforts during COVID-19 [@individual-pharmaceuticals].
```

]((images/ebmdatalab-trials-original.png){{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

> <!--To Do: Discussion of the absurd number of references?-->

Reviewer 2: Hopefully we can plot this! 🤞

This computational approach allows for the information in the manuscripts to be kept up to date automatically.

Beyond the immediate goal of applying Manubot to the challenges of COVID-19, we have also expanded Manubot to allow for broader participation in open publishing from fields where computational training in tools like version control is uncommon. 

> Several review articles on aspects of COVID-19 have already been published, including reviews on the disease epidemiology [@doi:10.1016/j.molmed.2020.02.008], immunological response [@doi:10.1016/j.immuni.2020.05.002], diagnostics [@doi:10.1126/scitranslmed.abc1931], and pharmacological treatments [@doi:10.1016/j.immuni.2020.05.002; @doi:10.1001/jama.2020.6019] and others that provide narrative reviews of progress on some important ongoing COVID-19 research questions [@doi:10.1038/d41591-020-00026-w; @doi:10.1001/jama.2020.12839].

Author: I pilfered this for the intro, will need to rework the flow here when I get to the discussion.
